lib/java/spring/web/src/main/java/co/airy/spring/web/filters/RequestLoggingFilter.java

```diff
@@ -31,6 +35,12 @@
     private final ObjectMapper objectMapper;
     private final RequestLoggingIgnorePatterns requestLoggingIgnorePatterns;
 
+    @Value("${core.id")
+    private String coreId;
+
+    @Autowired(required = false)
```

Comment:
As a principle we only use the `@Autowired` annotation in tests and the [constructor injection](https://reflectoring.io/constructor-injection/) everywhere else.